.travis.yml

```diff
@@ -14,6 +14,7 @@ php:
 before_install:
   - phpenv config-rm xdebug.ini || true
   - composer self-update
+  - if [ "$DEPENDENCIES" = "beta" ]; then composer config minimum-stability beta; fi;
```

Comment:
You need to update the matrix to include this `DEPENDENCIES` env-variable.

```
matrix:
    include:
        - php: '7.1'
          env: DEPENDENCIES=beta
```

Reply:
added

Reply:
whole line not needed anymore